djangoproject/templates/fundraising/index.html

```diff
@@ -198,6 +199,12 @@ <h2 id="who-are-the-django-fellows">Who are the Django Fellows?</h2>
 
       <p>Former Django Fellows:</p>
 
+      <p><strong><a href="https://github.com/carltongibson">Carlton Gibson</a>
+        (2018-2023)</strong> - a longtime Django user, core contributor to
+        Django REST Framework, maintainer of Django Filter and Django Crispy
+        Forms, and a contributor to many other packages in the Django ecosystem.
+        Carlton <a href="https://www.djangoproject.com/weblog/2018/jan/12/dsf-welcomes-carlton-gibson-its-newest-fellow/">
+        began as a part-time Fellow in January 2018</a>.</p>
```

Comment:
I'd add a sentence about retirement as we did for Tim, e.g. 
```suggestion
        Carlton began as a part-time Fellow in January 2018.
        In 2023 Carlton retired after five years of service.</p>
```